docs/src/design-system/color-palette/palette.tsx

```diff
@@ -65,8 +65,9 @@ const ColorPalette: React.FC = () => {
       <Grid
         container
         sx={{
-          bgcolor: palette.mode === 'dark' ? palette.background.paper : palette.logo,
-          color: palette.common.white,
+          bgcolor:
+            palette.mode === 'dark' ? palette.background.paper : palette.common.white,
```

Comment:
If I am not wrong, I think we can just remove `bgcolor` and `color` properties. We already have pre-defined light and dark modes of different `bgcolor` and `color`, so when we switch modes, the colors should be able to change based on the mode.

Reply:
You're right.  I've removed those in the most recent commit.  Thanks for pointing it out.